transform/transformers/CORATransformer.py

```diff
@@ -141,7 +141,7 @@ def comment(q, d):
             return '0' if q not in d else '1' if len(d[q].strip()) > 0 else '0'
 
     defn = [
-        (range(1, 4, 1), "1", Format.zeroone, Processor.constant),
+        (range(1, 4, 1), "0", Format.zeroone, Processor.false),
```

Comment:
The tests mention constant, but the method is now called false. They should probably be consistent but I dont mind which way it goes.